lib/ansible/modules/system/java_cert.py

```diff
@@ -218,6 +261,9 @@ def main():
     argument_spec = dict(
         cert_url=dict(type='str'),
         cert_path=dict(type='str'),
+        pkcs12_path=dict(type='str'),
```

Comment:
I believe, if `pkcs12_path` is path, then this will be of type 'path' rather than 'str'